README.md

```diff
@@ -1,235 +1,19 @@
-main_deployed_at_block = 100
-side_deployed_at_block = 101
-last_main_to_side_sign_at_block = 121
-last_side_to_main_signatures_at_block = 122
-last_side_to_main_sign_at_block = 122
-```
-
-**all fields are required**
-
-- `main_contract_address` - address of the bridge contract on main chain
-- `side_contract_address` - address of the bridge contract on side chain
-- `main_deployed_at_block` - block number at which main contract has been deployed
-- `side_deployed_at_block` - block number at which side contract has been deployed
-- `last_main_to_side_sign_at_block` - number of the last block for which an authority has relayed signatures to the side
-- `last_side_to_main_signatures_at_block` - number of the last block for which an authority has relayed signatures to the main
-- `last_side_to_main_sign_at_block` - number of the last block for which an authority has confirmed messages relayed to main
-
-### deployment and run
-
-[read our deployment guide](deployment_guide.md)
-
-### deposit
-
-![deposit](./res/deposit.png)
-
-### withdraw
-
-![withdraw](./res/withdraw.png)
-
-
-### considerations for relaying messages to main
-
-a bridge `authority` has to pay for gas (`cost`) to execute `Main.acceptMessage` when
-sending a message from the `side` chain to the `main` chain. When creating `BridgeRecipient`s, it is prudent to keep this cost in mind.
-
-parity-bridge connects a value-bearing ethereum blockchain `main`
-(initially the ethereum foundation chain)
-to a non-value-bearing PoA ethereum blockchain `side` (initially the kovan testnet).
-
-value-bearing means that the ether on that chain has usable value in the sense that
-in order to obtain it one has to either mine it (trade in electricity)
-or trade in another currency.
-non-value-bearing means that one can easily obtain a large amount of ether
-on that chain for free.
-through a faucet in the case of testnets for example.
-
-the bridge authorities *should also be* the validators of the `side` PoA chain.
-transactions by the authorities are therefore free (gas price = 0) on `side`.
-
-to execute a transaction on `main` a bridge authority has to spend ether to
-pay for the gas.
+[travis-image]: https://travis-ci.org/tomusdrw/parity-bridge.svg?branch=master
+[travis-url]: https://travis-ci.org/tomusdrw/parity-bridge
+[coveralls-image]: https://coveralls.io/repos/github/tomusdrw/parity-bridge/badge.svg?branch=master
+[coveralls-url]: https://coveralls.io/github/tomusdrw/parity-bridge?branch=master
 
-this opens up an attack where a malicious user could spam `Side.relayMessage`.
-It would cost the attacker no `main` chain wei and essentially
-free `side` testnet wei to cause the authorities to spend significant amounts of wei
-to relay the message to `main` by executing `Main.acceptMessage`.
-an attacker is able to exhaust bridge authorities funds on `main`.
+- bridge is able to pass arbitrary messages between an ethereum Proof of Authority chain and a Substrate Chain or Polkadot Parachain.
```

Comment:
```suggestion
- bridge is able to pass arbitrary messages between an Ethereum Proof of Authority chain and a Substrate Chain or Polkadot Parachain.
```

Correct me if I'm wrong, but a Parachain isn't necessarily a Substrate chain, so will this bridge actually work for those chains?

Reply:
Yeah, I meant a substrate-based parachain. Re-phrased a bit.